docs/setup.md

```diff
@@ -30,25 +30,33 @@ Follow this guide to setup your development machine.
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
+
+6. Creating user
```

Comment:
They need this as its hard to setup project for new user. Our documentation should be so brief that no one should be stuck in between while setting up the project